DataSetDocker/Dockerfile

```diff
@@ -0,0 +1,71 @@
+ARG BASE_CONTAINER=jupyter/minimal-notebook
+FROM $BASE_CONTAINER
+
+LABEL maintainer="Igor Gudich <igor.gudich@esss.se>"
+
+USER root
+
+# cmake and clang to compile source code
+RUN apt-get update && \
+    apt-get install -y cmake && \
+    apt-get install -y clang-6.0 && \
+    apt-get install -y mercurial && \
+    apt-get install -y libboost-all-dev && \
```

Comment:
Just trying to build this myself. This ends up installing many boost libraries that are not needed. Those then pull also things like OpenMPI... If we change this to `libboost-dev` instead of `libboost-dev`, is that still working for you?

Reply:
Do you mean `libboost-all-dev` to `libboost-dev`? I can do this, but anyway boost is deinstalled after building dataset, so it is not very important.